src/library/Uno.Material/Styles/Controls/Button.xaml

```diff
@@ -358,7 +356,7 @@
 									<Storyboard>
 										<DoubleAnimation Storyboard.TargetName="CommonStatesOverlay"
 														 Storyboard.TargetProperty="Opacity"
-														 Duration="0:0:0.25"
+														 Duration="0:0:1"
```

Comment:
Any particular reason for this to not follow suit? And, should we create a resource for "0:0:1"?